modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -588,9 +588,13 @@ definition {
 			ObjectAdmin.gotoRelationshipsTab();
 
 			ObjectCustomLayouts.addEntryOnRelationshipTab(selectEntry = "Entry Test B");
+
+			Alert.viewSuccessMessage();
```

Comment:
Why are we adding an assertion here to check for the alert message? Since this is the setup, we shouldn't be making assertions here.